src/audio/Kconfig

```diff
@@ -11,18 +12,13 @@ config COMP_BASEFW_IPC4
         help
           Select for BASEFW component
 
-config COMP_COPIER
-        bool "COPIER component"
-        default y
-        depends on IPC_MAJOR_4
-        help
-          Select for COPIER component
+rsource "copier/Kconfig"
 
 config HOST_DMA_RELOAD_DELAY_ENABLE
 	bool "Delay reloading DMA for host interfaces"
 	default y
 	help
-	  When deep buffers are used the firmware doesn't need to have the
+	  When deep buffers are used the firmware does not need to have the
```

Comment:
why make this change?

Reply:
because the" ' " makes code displays very strange.